raiden_contracts/tests/test_channel_withdraw.py

```diff
@@ -153,20 +152,20 @@ def test_withdraw_call_near_expiration(
     channel_identifier = create_channel_and_deposit(A, B, 10, 1)
     # The block must still be one block in the future when the transaction is
     # processed, so we have to choose an expiration two block in the future
-    expiration = web3.eth.block_number + 2
+    expiration = web3.eth.get_block("latest").timestamp + 30  # type: ignore
 
     (signature_A_for_A, signature_B_for_A) = create_withdraw_signatures(
         [A, B], channel_identifier, A, withdraw_A, expiration
     )
 
     call_and_transact(
         token_network.functions.setTotalWithdraw(
-            channel_identifier=channel_identifier,
-            participant=A,
-            total_withdraw=withdraw_A,
-            expiration_block=expiration,
-            participant_signature=signature_A_for_A,
-            partner_signature=signature_B_for_A,
+            channel_identifier,
+            A,
+            withdraw_A,
+            expiration,
+            signature_A_for_A,
+            signature_B_for_A,
```

Comment:
Just for knowledge sharing: why did this work in the past and now caused all these signature mismatch errors? :thinking:
EDIT: below for the `firstTimestampAllowedToMonitorChannel` function call it works again...

Reply:
I don't know why this stopped working. Maybe @palango could have some explanation? Maybe something is missing to link the python call to the solidity signature?